homeassistant/components/osoenergy/const.py

```diff
@@ -0,0 +1,66 @@
+"""Constants for OSO Energy."""
+from typing import Any
+
+from homeassistant.const import STATE_OFF, STATE_ON, Platform
+
+ATTR_UNTIL_TEMP_LIMIT = "until_temp_limit"
+ATTR_V40MIN = "v40_min"
+CONFIG_ENTRY_VERSION = 1
+DOMAIN = "osoenergy"
+
+EXTRA_HEATER_ATTR: dict[str, dict[str, Any]] = {
```

Comment:
It should be moved to `water_heater.py`. You should declare only constants here, which are used in multiple modules.